training/nvidia-bootc/Containerfile

```diff
@@ -146,6 +148,9 @@ RUN mv /etc/selinux /etc/selinux.tmp \
     # Install rhc connect for insights telemetry gathering
     && . /etc/os-release && if [ "${ID}" == "rhel" ]; then \
         dnf install -y rhc rhc-worker-playbook; \
+        sed -i -e "/^VARIANT=/ {s/^VARIANT=.*/VARIANT=\"RHEL AI\"/; t}" -e "\$aVARIANT=\"RHEL AI\"\"" /etc/os-release;
```

Comment:
amd, intel?

Reply:
Will do in next PR, in anycase GA is nvidia.